akka-stream/src/main/scala/akka/stream/impl/io/TcpListenStreamActor.scala

```diff
@@ -13,7 +13,7 @@ import akka.io.Tcp._
 import akka.stream.{ FlowMaterializer, ActorFlowMaterializerSettings }
 import akka.stream.impl._
 import akka.stream.scaladsl.Flow
-import akka.stream.scaladsl.StreamTcp
+import akka.stream.scaladsl.{ Tcp ⇒ StreamTcp }
```

Comment:
yes, we use both in this file, could have renamed the other, but whatever this is internal